app/components/shared/rejection_reasons/component_helper.rb

```diff
@@ -0,0 +1,22 @@
+module RejectionReasons::ComponentHelper
```

Comment:
I've reworked this helper https://github.com/DFE-Digital/apply-for-teacher-training/pull/6723/commits/6722c70972b3a1c51be41d21872bb548f030c1fb so that the constructor / attr reader logic all lives in a parent class. DRYs things up a bit at the cost of an ancient inheritance pattern :D